packages/react-components/react-virtualizer/src/components/Virtualizer/Virtualizer.types.ts

```diff
@@ -20,7 +20,7 @@ export type VirtualizerSlots = {
   afterContainer: NonNullable<Slot<'div', 'tr'>>;
 };
 
-export type VirtualizerState = ComponentState<VirtualizerSlots> & {
+export type IVirtualizerState = {
```

Comment:
```suggestion
export type VirtualizerState = {
```

`I` interface prefixes are not used in this repository

Reply:
Mostly used this to define internal state vs exported state with slots - should we convert to _?
`
export type VirtualizerState = ComponentState<VirtualizerSlots> & IVirtualizerState;
`

Reply:
well in thise case, why do you need a separate type ?
```ts
export type VirtualizerState = ComponentState<VirtualizerSlots> & {
  /* Define the rest of stat here */
}
```

Reply:
We extend these base values in the scroll wrapper classes, we import it there.

Reply:
Updated with 'VirtualizerConfigState' / 'VirtualizerConfigProps'